client/src/screens/events/Detail.js

```diff
@@ -263,16 +354,20 @@ class Detail extends Component {
         }}
             style={styles.map}
           >
-            {mapResponseMarkers.map(marker => (
+            <MyLocationMarker
+              myPosition={this.state.myPosition}
+            />
+            {this.state.responseMarkers && this.state.responseMarkers.map(marker => (
               <Marker coordinate={marker} key={marker.id}>
                 <UserMarker
                   name={marker.displayName}
                   status={marker.status}
                   destination={marker.destination}
+                  locationTime={`Updated ${marker.locationTime}`}
                 />
               </Marker>
         ))}
```

Comment:
indentation. The closing line should be indented as much as the outermost opening paren/brace

Reply:
ty. the linter cant see these for some reason